programs/mango-v4/tests/program_test/mango_client.rs

```diff
@@ -617,7 +617,7 @@ impl ClientInstruction for TokenWithdrawInstruction {
 
 pub struct TokenDepositInstruction {
```

Comment:
possibly we should make a `SimpleTokenDepositInstruction` that doesn't need the reduce_only and bank_index args. Or maybe we implement defaults and do `...Default::default()` everywhere. But not in this PR I'd say.